applications/humanres/widget/RecruitmentScreens.xml

```diff
@@ -249,7 +249,7 @@ under the License.
                                         <include-form name="FindApprovals" location="component://humanres/widget/forms/RecruitmentForms.xml"/>
                                     </decorator-section>
                                     <decorator-section name="search-results">
-                                        <include-form name="ListApprovals" location="component://humanres/widget/forms/RecruitmentForms.xml"/>
+                                        <include-grid name="ListApprovals" location="component://humanres/widget/forms/RecruitmentForms.xml"/>
```

Comment:
When testing with data from .`/gradlew loadAll`, plus some job requisitions created while testing earlier screens, I get the following error when searching from https://localhost:8443/humanres/control/FindApprovals.

Note: This error is not related to your PR, @PierreSmits , just mentioning it in case you knew the cause.


```
2022-01-21 15:38:55,042 |sse-nio-8443-exec-16 |ControlServlet                |E| Error in request handler:
java.lang.RuntimeException: Error rendering included form named [ListApprovals] at location [component://humanres/widget/forms/RecruitmentForms.xml]: java.lang.NullPointerExceptionjava.lang.NullPointerException
```